src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepProcess.java

```diff
@@ -118,21 +137,28 @@ private void captureOutput(Process process, String vepOutput) {
     }
 
     public void write(byte[] bytes) throws IOException {
```

Comment:
Could a class be created to test the workflow for the methods (write, flush, close...) that check if a process is open?